airdrop-cli/src/commands/tally-multi-csv.ts

```diff
@@ -3,14 +3,14 @@ import { fetchPublicRepositories, processRepo } from "../invoke";
 import { loadingBar, writeCSV } from "../utils";
 
 @command({
```

Comment:
I had no idea that `@` is valid syntax. This code is interesting!

Reply:
It's a decorator, often times they throw errors depending on build settings. Pretty sure they are an older thing you rarely see them anywhere anywhere